install_ansible_for_testing.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# this is used to install ansible into a venv or tox env
+# for testing purposes
```

Comment:
I think it should be moved into some subdirectory (like .travis/ or tests/) if it is only testing-related. Ditto for the next file.